crates/node/builder/src/launch/common.rs

```diff
@@ -371,12 +370,14 @@ where
             assert_ne!(unwind_target, PipelineTarget::Unwind(0), "A static file <> database inconsistency was found that would trigger an unwind to block 0");
 
             info!(target: "reth::cli", unwind_target = %unwind_target, "Executing an unwind after a failed storage consistency check.");
+            
+            let (_tip_tx, tip_rx) = watch::channel(B256::ZERO);
```

Comment:
unsure if this is correct, but it feels hacky, cc @Rjected

Reply:
I mean, continuous was somewhat hacky too, the pipeline is only being used for an unwind here, so I'm not sure either case has predictable behavior if we somehow start going forward instead. We may also want to improve provider initialization s.t. you could build an unwind only provider without doing this, or adding noop downloaders etc

Reply:
I can work on this.

Reply:
let's do that in a follow up instead